demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1242,6 +1244,94 @@ def get_general_paths(path):
     return files
 
 
+class IDSetType(Enum):
+    PLAYBOOK = 'playbooks'
+    INTEGRATION = 'integrations'
+    SCRIPT = 'scripts'
+    TEST_PLAYBOOK = 'TestPlaybooks'
+    WIDGET = 'Widgets'
+    CLASSIFIER = 'Classifiers'
+    MAPPER = 'Mappers'
+    REPORT = 'Reports'
+    DASHBOARD = 'Dashboards'
+    INCIDENT_FIELD = 'IncidentFields'
+    INCIDENT_TYPE = 'IncidentTypes'
+    INDICATOR_FIELD = 'IndicatorFields'
+    INDICATOR_TYPE = 'IndicatorTypes'
+    LAYOUTS = 'Layouts'
+
+    @classmethod
+    def has_value(cls, value):
+        return value in cls._value2member_map_
+
+
+class IDSet:
+    def __init__(self, id_set_dict={}):
+        self._id_set_dict = id_set_dict
+
+    def get_dict(self):
+        return self._id_set_dict
+
+    def get_list(self, item_type):
+        return self._id_set_dict.get(item_type, [])
+
+    def add_to_list(self, object_type: IDSetType, obj):
+        if not IDSetType.has_value(object_type):
+            raise ValueError(f'Invalid IDSetType {object_type}')
+
+        if object_type not in self._id_set_dict:
+            self._id_set_dict[object_type] = []
+
+        self._id_set_dict[object_type].append(obj)
+
+
+def merge_id_sets_from_files(first_id_set_path, second_id_set_path, output_id_set_path, print_logs: bool = True):
+    """
+    Merges two id sets. Loads them from files and saves the merged unified id_set into output_id_set_path.
+    """
+    with open(first_id_set_path, mode='r') as f1:
+        first_id_set = json.load(f1)
+
+    with open(second_id_set_path, mode='r') as f2:
+        second_id_set = json.load(f2)
+
+    unified_id_set, has_duplicates, duplicates = merge_id_sets(first_id_set, second_id_set, print_logs)
+
+    if unified_id_set:
+        with open(output_id_set_path, mode='w', encoding='utf-8') as f:
+            json.dump(unified_id_set.get_dict(), f, indent=4)
+
+    return unified_id_set, has_duplicates, duplicates
+
+
+def merge_id_sets(first_id_set_dict: dict, second_id_set_dict: dict, print_logs: bool = True):
```

Comment:
why not join both id sets and then use the original `has_duplicate` function?

Reply:
I remember we had some duplicates in our id_sets. So I wanted to avoid the noice. To represent only the duplicates that were added from one of them. But if we don't have these duplicates anymore, then I can merge them

Reply:
ok, then leave it as is